pkgs/by-name/do/docuseal/web.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchYarnDeps
+, yarn
+, fixup_yarn_lock
+, nodejs
+, which
+, version
+, src
+, meta
+, rubyEnv
+}:
+stdenv.mkDerivation {
+  pname = "docuseal-web";
+  inherit version src meta;
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-Te20zmX8Kdme0ypJi39LbxmuPqnwQJHZAWWMryQTEcw=";
+  };
+
+  nativeBuildInputs = [
+    yarn
+    fixup_yarn_lock
```

Comment:
You [should use `fixup-yarn-lock` instead](https://github.com/NixOS/nixpkgs/issues/240174).